service-perfplat/setup.py

```diff
@@ -0,0 +1,46 @@
+from os import path
+
+from setuptools import setup
+
+setup(
+    name='perfplat',
+    version='0.1.0',
+    description='Performance platform background worker and API',
+    long_description='',
+    long_description_content_type='text/plain',
+    packages=[
+        'perfplat',
+    ],
+    python_requires='>=3.7, <4',
+    install_requires=[
+        'SQLAlchemy',
+        'psycopg2',
+        'Flask',
+        'flask-lambda',
+        'flask-lambda-python36',
```

Comment:
Do we need both of these lines for flask-lambda?

Reply:
Nope. Removed!